docs/release.md

```diff
@@ -28,24 +28,90 @@ cd backplane-cli
 git remote add upstream https://github.com/openshift/backplane-cli.git
 ```
 
+### Determine a new version
+backplane-cli follows [semver](https://semver.org/). The version is in format `v[Major].[Minor].[Patch]`.
+
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backward compatible manner
+- PATCH version when you make backward compatible bug fixes
+
+Review the commits since last release:
+```
+$ git fetch upstream
+
+$ git log $(git describe --tags --abbrev=0 upstream/main)..upstream/main --pretty=format:"%h %s" |
+gawk '
+{
+  # Extract the commit message without the hash and ticket prefix
+  # Pattern: hash + space + [ticket] + space + type + ":"
+  # Example: "a1b2c3d [PROJ-123] feat: some message"
+
+  # Remove the hash and space
+  msg = substr($0, index($0,$2))
+
+  # Regex to extract type: after "] " followed by letters, colon
+  if (match(msg, /\] *([a-z]+):/, m)) {
+    type = m[1]
+    groups[type] = groups[type] "\n- " $0
+  } else {
+    groups["others"] = groups["others"] "\n- " $0
+  }
+}
+END {
+  order = "feat fix chore docs test others"
+  split(order, o)
+  for (i in o) {
+    t = o[i]
+    if (t in groups) {
+      print "## " toupper(substr(t,1,1)) substr(t,2)
+      print groups[t] "\n"
+    }
+  }
+}
+'  > /tmp/release-note.md
+```
+This saves the grouped commits in `/tmp/release-note.md`, please review and modify the release note file.
+
+Increase `Major` when:
+- It has changes that breaks backward compatibility:
+    - For example, refactoring the CLI which changes the subcommand and argument format.
+    - The user need to use a new command to perform a task which the old command no longer works.
```

Comment:
I feel this kind of change can be considered as Minor?

Major changes to me, more like the server side has significant change which the client side needs to adapt.